lightkurve/search.py

```diff
@@ -360,10 +360,34 @@ def _fetch_tesscut_path(self, target, sector, download_dir, cutout_size):
 
         # Resolve SkyCoord of given target
         coords = _resolve_object(target)
-        cutout_path = TesscutClass().download_cutouts(coords, size=cutout_size,
-                                                      sector=sector, path=tesscut_dir)
 
-        path = os.path.join(download_dir, cutout_path[0][0])
+        # build path string name and check if it exists
+        # this is necessary to ensure cutouts are not downloaded multiple times
+        sec = TesscutClass().get_sectors(coords)
+        sector_name = sec[sec['sector'] == 1]['sectorName'][0]
+        if isinstance(cutout_size, int):
+            size_str = str(int(cutout_size)) + 'x' + str(int(cutout_size))
+        elif isinstance(cutout_size, tuple) or isinstance(cutout_size, list):
+            size_str = str(int(cutout_size[0])) + 'x' + str(int(cutout_size[1]))
+
+        matchstring = r"{}_{}*_{}*_{}_astrocut.fits".format(sector_name,
```

Comment:
@nksaunders This looks like the line that will determine how closely an existing file has to match, e.g. how similar the ra/dec values should be.  I think it may be a good idea to be very explicit here as to how close we want the match to be, e.g. am I correct to think that using `{:.3f}` means we require a match within 3.6 arcseconds?  (If that's correct, let's make sure we add it as a comment in the code.)  
```suggestion
        matchstring = r"{}_{:.3f}*_{:.3f}*_{}_astrocut.fits".format(sector_name,
```